### DISCUSSION TITLE
Hide Navbar in certain pages.

### DISCUSSION THREAD

**Discussion by shadowoff09** [vercel/next.js#60849] [Help]
### Summary

Hey devs, i have in the layout of the root directory an navbar, so every page have thta navbar, until then everyting is good, but in /home i want to replace the navbar with sidebar, for that i create an layout.tsx for the /home directory and put the sidebar there, but now the navbar and the sidebar appear, how can i when im at /home, hide the navbar and just show the sidebar?

**Comment by ezeparziale** [comment]
You need to use [route-groups](https://nextjs.org/docs/app/building-your-application/routing/route-groups)

Example:
![Captura de pantalla 2024-01-19 a la(s) 09 04 28](https://github.com/vercel/next.js/assets/82482391/58c00c9b-9e12-454e-817e-a8a59ee29385)



home layout:

```
export default function HomeLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <>
      <p>Home Layout</p>
      <div>{children}</div>
    </>
  );
}
```


site layout:


```
export default function HomeLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <>
      <p>Site Layout</p>
      <div>{children}</div>
    </>
  );
}
```

global layout:

```
import type { Metadata } from "next";
import { Inter } from "next/font/google";
import "./globals.css";

const inter = Inter({ subsets: ["latin"] });

export const metadata: Metadata = {
  title: "Create Next App",
  description: "Generated by create next app",
};

export default function RootLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <html lang="en">
      <body className={inter.className}>{children}</body>
    </html>
  );
}
```
